### ISSUE TITLE
I see redundant images after imagenet_export and draw_ponder_maps

### ISSUE THREAD

**Issue by ahyunSeo** [mfigurnov/sact#8]
I ran the whole validation set with batch size 50, so I got 50000 image jpgs and 50000 ponder map pngs.
I tried to find the figure in your README just to make sure everything's done well.
I only found the one below,
![image](https://user-images.githubusercontent.com/22852018/39346328-842d5a74-4a28-11e8-8305-12ffa408cc52.png)
and I found I have couple of redundant images like:
![screenshot from 2018-04-27 14-15-12](https://user-images.githubusercontent.com/22852018/39346264-23e7d0e0-4a28-11e8-825c-7e58a2c9b1d3.png)
My ImageNet raw-data seems find but I think I have a problem with tfrecord, 
since I get almost 0 evaluation accuracy only in ImageNet (doing well on cifar-10).
I also posted an issue about the ImageNet script.
https://github.com/tensorflow/models/issues/4051
I'm planning to reproduce part of your code to pytorch, 
and if my issue is just because of the tfrecord issue, I will be fine.
Or am I missing something?
I use 2 Titan XP with python 2.7

**Comment by mfigurnov** [comment]
It does look that your data preprocessing is incorrect and you are reading the same images multiple times. Make sure the data you have previously downloaded matches the format expected by the preprocessing script. It is important that the data ends up in correct folders.

**Comment by ahyunSeo** [comment]
Thanks again!
